defaults/main.yml

```diff
@@ -40,4 +40,14 @@ elasticsearch_default_password: changeme
 
 elasticsearch_password: CHANGEME
 elasticsearch_min_heap_size: 2g
-elasticsearch_max_heap_size: 2g
\ No newline at end of file
+elasticsearch_max_heap_size: 2g
+
+# Offline installation
+elasticsearch_offline_install: false
+elasticsearch_baseurl: 
+elasticsearch_gpg_key_path: "{{ playbook_dir }}/files/groups/elasticsearch/key"
+elasticsearch_gpg_key: ELASTICSEARCH-GPG
+elasticsearch_yum_repo_template: elasticsearch.repo.j2
+
+# enable in case if you change data path & log path
+elasticsearch_change_path: false
```

Comment:
maybe use `elasticsearch_change_install_path` for better clarity or change the logic to use install path as variable and run related task if that variable is not empty.